include/pel_handler.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+namespace vpd
+{
+/**
+ * @brief Class to support logging PEL.
+ *
+ * This class provides framework to call 'create' method of 'phosphor-logging'
+ * service to log a PEL.
+ */
+class PelHandler
+{
+  public:
+    using PelData = std::unordered_map<std::string, std::string>;
+
+    static constexpr auto VpdDefaultErrorIntf =
```

Comment:
Why can't be used from constants?

Reply:
as we have introduced new class,  thought it should be appropriate keep all related things in this class only.